xmtp/src/storage/encrypted_store/models.rs

```diff
@@ -36,6 +42,7 @@ pub struct DecryptedMessage {
     pub convo_id: String,
     pub addr_from: String,
     pub content: Vec<u8>,
+    pub state: i32,
```

Comment:
For Clarity: Am I correct in that DecryptedMessage is going to be used for local never encrypted messages? Outbound messages are also going be DecryptedMessages? Nothin against this, just attempting to stay in the loop